<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btNewDev">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Repaint</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tlServer">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>anka-tango3.ka.fzk.de:10000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbServer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDevice">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Attribute</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tlAttr">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>testImage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btChangeDevice">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>AddDevice</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDevice_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tlDevice">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sys/tg_test/mytest</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbState">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>20</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>it's work</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btScale">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCurWin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CurWin_0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btMkSnapshot">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Make snapshot</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCurWorkiningDev">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cur Dev: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
